src/htmx.js

```diff
@@ -1081,6 +1085,24 @@ return (function () {
             getInternalData(elt).sseEventSource = source;
         }
 
+        function processSSEForward(elt, sseEventName) {
```

Comment:
Rather than creating a new method, what about renaming this to something like `processSSEClient` and passing in a swap boolean?  That would reduce some of the code duplication and is about as clear IMO.

Reply:
Yup. I agree with that as well.

Reply:
I've made the change.